regulations/templates/regulations/node.html

```diff
@@ -14,6 +14,20 @@
 {% endif %}
 </p>
 {% endif %}
+
+<div class="comment-wrapper" data-section="{{ node.markup_id }}">
+  <div class="toggle">(click to toggle comment box)</div>
+  <div class="comment">
+    <h3>Enter your comment below:</h3>
+    <form>
+      <textarea role="textbox" aria-multiline="true"></textarea>
+      <div class="queued"></div>
+      <input type="file">
+      <button type="submit">Submit</button>
+    </form>
+  </div>
+</div>
```

Comment:
I think that'd work! Unfortunately, I used caps when first loading the data... You'll have to update https://github.com/18F/regulations-site/blob/master/regulations/generator/html_builder.py#L125 to lower-case the node_type, I think. The next set of data should be lower-cased.